lib/logger/lib/logger.ex

```diff
@@ -191,6 +191,11 @@ defmodule Logger do
       should be logged. Defaults to `false`. This option only has an effect
       if `:handle_otp_reports` is true.
 
+    * `:metadata` - global primary metadata to be included in all log messages.
+      Defaults to `[]`. See [the configuration
+      section](https://www.erlang.org/doc/man/kernel_app.html#configuration) of
+      the documentation for the `kernel` Erlang application.
```

Comment:
```suggestion
    * `:metadata` - global primary metadata to be included in all log messages.
      Defaults to `[]`. This can be overridden at the process level with `metadata/1`
      or each on log call as desired.
```

Reply:
We should also add a note to `Logger.metadata/0` that it only returns the process metadata and not the global one.

Let's also add a test in `logger_test`. You will already find some tests that call `App.stop` and `Application.start(:logger)` to test the level is read correctly. We can do similar for metadata.

Reply:
Done and done 👍